monai/transforms/spatial/array.py

```diff
@@ -2462,3 +2463,79 @@ def __call__(
         if not self._do_transform:
             return img
         return self.grid_distortion(img, distort_steps=self.distort_steps, mode=mode, padding_mode=padding_mode)
+
+
+class Split(Transform):
```

Comment:
As we are adding this transform to the core transforms, I feel `Split` may be too general a name.
How about `GridSplit` or `SplitPatches`?
@ericspod @rijobro What do you guys think?

Thanks.

Reply:
Thanks for the feedback.